src/errors.rs

```diff
@@ -54,6 +54,17 @@ pub enum Error {
     /// Network error occurring at Vault level which has no bearing on clients, e.g. serialisation
     /// failure or database failure
     NetworkOther(String),
+    /// While parsing or converting, precision would be lost.
+    LossOfPrecision,
+    /// The safecoin amount would exceed
+    /// [the maximum value for `Coin`](constant.MAX_COINS_VALUE.html).
```

Comment:
This link will be broken if we don't expose `MAX_COINS_VALUE` in lib.rs.

Also, `Coin` should be `Coins`.